### PR TITLE
Quicktext hu (magyar - Hungarian) translation

### PR DESCRIPTION
**Pull request by ovari**

### CODE REVIEW

File: install.rdf [hunk 1]
```diff
@@ -8,10 +8,11 @@
     <em:name>Quicktext</em:name>
     <em:version>1.1</em:version>
     <em:unpack>true</em:unpack>
-    <em:description>Adds a toolbar with unlimited number of text to quickly insert. It's also possible to use variables like [[TO=firstname]]. With settings for everything.</em:description>
```
Comment (jobisoft) [inline install.rdf]: Please undo this Change, not related to adding hu translation.

**Review by jobisoft — CHANGES_REQUESTED** [review]
Please undo changes not related to hu translation

File: install.rdf (hunk 1 above)
Comment (jobisoft) [reply install.rdf]: This is the original description from Emil, I would like to keep it for now

Comment (jobisoft) [reply install.rdf]: The fixes for URLs and links are ok, but I want to keep Emil's original description.

Comment (ovari) [reply install.rdf]: Should be done.

**Comment by ovari** [comment]
Is it ready for to merge? Thank you

**Review by jobisoft — APPROVED** [review]

**Comment by jobisoft** [comment]
Thanks!

**Comment by ovari** [comment]
1. Does the version in the install.rdf need to be increased before uploading to [ATN](https://addons.thunderbird.net/hu/thunderbird/addon/quicktext/)?

2. Please update https://addons.thunderbird.net/hu/thunderbird/addon/quicktext/ : 
Name: `Gyorsszöveg (Quicktext)`
Summary: `Eszköztár, parancsfájlok, sablonok a korlátlan lehetőségekhez, beleértve a [[Címzett=Keresztnév]] változókat is, a szöveg gyors beillesztéséhez`
Image captions:
a. from `A sample how it could look` to `Példakép`
b. from `The variables menu` to `A változók menü`
Description: Leave as is

Thank you

**Comment by jobisoft** [comment]
Sadly I cannot upload that new version to ATN. I will release one here on Github. The new version will not pass ATN review. The current version has passed review by grace and some stuff has to be changed before the next verison may be uploaded.

**Comment by ovari** [comment]
Can you update https://addons.thunderbird.net/hu/thunderbird/addon/quicktext/ ?

**Comment by jobisoft** [comment]
No I cannot, the current version will not pass review (done by ATN reviewers).

**Comment by ovari** [comment]
We are talking about the `name`, `summary` and `image captions`, not the xpi file.

**Comment by jobisoft** [comment]
yes, that I can do, wanna post it here?

**Comment by ovari** [comment]
Please see:
https://github.com/thundernest/quicktext/pull/37#issuecomment-432824891

**Comment by jobisoft** [comment]
That comment did not have that text in the initial version...

**Comment by jobisoft** [comment]
Done

**Comment by ovari** [comment]
Sorry and thank you!

**Pull request merged**